kratos/includes/condition.h

```diff
@@ -235,7 +235,7 @@ class Condition : public GeometricalObject, public Flags
      * @return a Pointer to the new condition
      */
     virtual Pointer Create(IndexType NewId, NodesArrayType const& ThisNodes,
-			   PropertiesType::Pointer pProperties) const
+               PropertiesType::Pointer pProperties) const
     {
         return Kratos::make_shared<Condition>(NewId, GetGeometry().Create(ThisNodes), pProperties);
```

Comment:
The idea proposed in the technical committee meeting was to force derived classes to implement the Create and Clone methods. Ideally, you would not implement them in the base class (`virtual Pointer Create(...) = 0;`). However, I think we would produce quite a few compilation errors if you just do that. An alternative is to make those methods throw an error now.

Reply:
We can do that in a new PR, many people should change that

Reply:
I would still throw an error, this will not break compilation and will clearly indicate that there is a bug somewhere (since there is no-one should be creating base Element and Condition instances now).

Reply:
I agree, but don't assume that, I remember that some months ago I found that many elements in the structural just have one of the Creates implemented, probably there are still elements with just one constructor

Reply:
I know, but that's precisely why I think that having it throw an error is better than silently creating base class instances.

Reply:
Well I will do it, prepare for failing tests, hehe